ran-core/src/main/java/io/ran/TestDoubleStore.java

```diff
@@ -0,0 +1,58 @@
+package io.ran;
+
+import javax.inject.Inject;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+public class TestDoubleStore<K, V> {
+	private final ConcurrentMap<K, V> map = new ConcurrentHashMap<>();
```

Comment:
Map on the left

Reply:
[54c5092](https://github.com/persequor-com/ran/pull/30/commits/54c5092f1d6512d675fab78fc5a76a15edd046cc)